### PR TITLE
Remove myself as SIG Cluster Lifecycle TL

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
I recently stepped into a new role and am no longer able to fulfill my duties as SIG Tech Lead. After discussing this with @vincepri @fabriziopandini @neolit123 @justinsb, I've decided to step down from the Tech Lead role for the time being. I still plan to be involved in the community and hope to be able to contribute to other SIGs as well in the future.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by aojea** [comment]
Good luck Cecile on your next adventure and thanks for all the great work you have done

**Review by neolit123 — COMMENTED** [review]
thanks for helping us with our chaotic group @CecileRobertMichon ! have fun at the next adventure.
/approve

File: sigs.yaml [hunk 1]
```diff
@@ -1026,9 +1026,6 @@ sigs:
       name: Vince Prignano
       company: Red Hat
     tech_leads:
-    - github: CecileRobertMichon
```
Comment (fabriziopandini) [inline sigs.yaml:1029]: @CecileRobertMichon given the outstanding work you did in our community, I think that you should add yourself to emeritus_leads

Comment (CecileRobertMichon) [reply sigs.yaml:1029]: I missed this was a thing! Done

**Review by pacoxu — APPROVED** [review]

**Pull request merged**